components/resources/omnibox_strings.grdp

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<grit-part>
+  <message name="IDS_OMNIBOX_OPEN_HERE_HINT"
+    desc="The button text contents to say that this suggestion will open the link in this tab, rather than switching to an existing one.">
+    Open here
+  </message>
+  <message name="IDS_ACC_OPEN_HERE_SUFFIX"
+    desc="Suffix for open here suggestions to explain keystroke used to open here.">
+    <ph name="TAB_SWITCH_SUFFIX">$1<ex>Brave http://brave.com bookmark</ex></ph>, currently open,
+    press Tab then Enter to switch to the open tab </message>
+  <message name="IDS_ACC_OPEN_HERE_BUTTON" desc="Announcement when open here button focused.">
+    Open here button, press Enter to open this url in the current tab, instead of switching to the
+    existing tab
+  </message>
```

Comment:
Are these switched? `Enter` should be to switch to the open tab and `Tab + Enter` to open here, no?
Also, maybe we should say `currently open in another tab`?

Reply:
I think these are correct:
1. `IDS_ACC_OPEN_HERE_SUFFIX` is a suffix for the AutocompleteMatch, which tells the user how to open the result in the current tab, if they want to (press tab, to select the pill, then press enter)
2. `IDS_ACC_OPEN_HERE_BUTTON` is the text if the button is focused. It explains what to do, and how to trigger it.